src/pages/AboutUs/styles.module.css

```diff
@@ -0,0 +1,173 @@
+.aboutUs {
+    width: 100vw;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 8%;
+}
+
+.firstStepAboutUS {
+    width: 80vw;
+    display: flex;
+    justify-content: space-between;
+    gap: 80px;
+
+}
+
+.firstStepContentWrapper {
+    width: 60%;
```

Comment:
Existe uma diferença de altura entre o texto ao lado e o grafo, com um padding em cima e embaixo isso é resolvido (o valor de 10vh pode ser ajustado, foi só um exemplo).
![image](https://user-images.githubusercontent.com/61962950/199133852-59e1dcaa-b75f-45e8-b443-4c5b3de81d3c.png)

```suggestion
    width: 60%;
    padding: 10vh 0;
```